Framework/Core/include/Framework/ParallelContext.h

```diff
@@ -0,0 +1,35 @@
+// Copyright CERN and copyright holders of ALICE O2. This software is
+// distributed under the terms of the GNU General Public License v3 (GPL
+// Version 3), copied verbatim in the file "COPYING".
+//
+// See http://alice-o2.web.cern.ch/license for full licensing information.
+//
+// In applying this license CERN does not waive the privileges and immunities
+// granted to it by virtue of its status as an Intergovernmental Organization
+// or submit itself to any jurisdiction.
+
+#ifndef FRAMEWORK_PARALLELCONTEXT_H
+#define FRAMEWORK_PARALLELCONTEXT_H
+
+namespace o2 {
+namespace framework {
+
+class ParallelContext {
```

Comment:
For better readability I wish to have some class documentation, how this can be used and also a reference to the service registry.

Reply:
Ok, I will try to improve documentation a bit.